frontend/__tests__/routes/_app+/letters+/_index.test.tsx

```diff
@@ -50,4 +50,28 @@ describe('Letters Page', () => {
       expect(data.letters[2].issuedOn).toBeUndefined();
     });
   });
+  it('retrieves letter types', async () => {
```

Comment:
It's fine here because the service is mocked. You don't see it in this test because it isn't in the PR's diff.

``` typescript
vi.mock('~/services/interop-service.server', () => ({
  getInteropService: vi.fn().mockReturnValue({
    getAllLetterTypes: vi.fn().mockResolvedValue([
      { code: 'ACC', nameEn: 'Accepted', nameFr: '(FR) Accepted' },
      { code: 'DEN', nameEn: 'Denied', nameFr: '(FR) Denied' },
    ]),
  }),
}));
```